include/mbedtls/entropy.h

```diff
@@ -181,6 +181,9 @@ int mbedtls_entropy_add_source( mbedtls_entropy_context *ctx,
  * \brief           Trigger an extra gather poll for the accumulator
  *                  (Thread-safe if MBEDTLS_THREADING_C is enabled)
  *
+ * \note            Depending on the platform (including kernel version), this
```

Comment:
“including kernel version” is both too much and too little information. It's only relevant for Linux, and even on Linux the kernel version is not the only relevant thing, but also the availability of a saved RNG seed and of a hardware RNG. Conversely, it is not meaningful on platforms that aren't based on a Linux kernel.

I think we should remove ”including kernel version“, and instead put a reference to a single place (either the module documentation, or the documentation of one function) with a longer discussion that covers both software and hardware.